app/views/shared/_item_card.html.erb

```diff
@@ -4,7 +4,7 @@
       <%= image_tag item.image_path %>
     </a>
     <div class="card-action">
-      <span class="card-title"><a href="/items/<%= item.id %>"><%= item.title %></a></span>
+      <span class="card-title"><a id="title-links" class="black-text" href="/items/<%= item.id %>"><%= item.title %></a></span>
```

Comment:
If something is going to be displayed on the page more than once, it shouldn't have an 'id' because that id will be reused. IDs are meant to refer to exactly ONE thing on a page, so if you want to style this with something like "title-links" make that a class.